omod/src/main/webapp/uiError.jsp

```diff
@@ -1,12 +1,7 @@
-<%@ include file="/WEB-INF/template/include.jsp"%>
-<%@ include file="/WEB-INF/template/header.jsp"%>
-
 <h1>UI Framework Error</h1>
```

Comment:
I'm talking about testing it without the legacy ui module since this is supposed to work without it. I guess I can add a controller for a some random endpoint that I enter in the browser and I call WebModuleUtil.stopModule from the controller.